src/cmd-cloud-prune

```diff
@@ -141,13 +143,13 @@ def main():
                         # S3().delete_object(args.bucket, f"{args.prefix}{str(current_build.id)}")
                     case "images":
                         raise NotImplementedError
-            if not args.dry_run:
-                build.setdefault("policy-cleanup", []).append("cloud-uploads")
-                builds_json_data["builds"][index] = build
+            build.setdefault("policy-cleanup", []).append("cloud-uploads")
+            builds[len(builds) - 1 - index] = build
```

Comment:
Hmm, that shouldn't be necessary. `build` is the original object within `builds` already.